drivers/sensor/bmi160/bmi160.c

```diff
@@ -18,387 +18,11 @@
 
 #include "bmi160.h"
 
-LOG_MODULE_REGISTER(BMI160, CONFIG_SENSOR_LOG_LEVEL);
+LOG_MODULE_DECLARE(BMI160, CONFIG_SENSOR_LOG_LEVEL);
```

Comment:
This commit sounds like it should just be code movement, but there are a few different style/convention changes as well. In the following commit, I similarly see a conversion from `#include <foo.h>` to `#include <zephyr/foo.h>`, which seems out of scope. Can stuff like that be its own commit, so that this commit is more obviously just code movement, and the next commit is more obviously just developing the v2 implementation?